frontend/pages/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+ font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+ background-color: black;
+margin: 0;
+overflow: hidden;
+}
+
+#title_text {
+    color:white;
+    text-align: center;
+    margin: 0 auto;
+    padding: 25px 50px 75px 100px;
+    font-size: 100px;
+    text-shadow: 5px 5px 10px red;
+    position: relative;
+    z-index: 3;
+  
+  
+}
+
+#Start_Button {
+    background-color: green;
+    
+    text-align: center;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    font-size: 50px;
+    cursor: pointer;
+    border-radius: 10px;
+    padding: 7.5px 15px;
+    margin-bottom: 10px;
+    transition: opacity 0.5s ease; /* Smooth transition */
+
+    
+    
+    
+}
+#Credits_Button {
+    background-color: red;
+    
+    text-align: center;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 10px auto;
+    font-size: 50px;
+    cursor: pointer;
+    border-radius: 10px;
+    padding: 7.5px 15px;
+    margin-bottom: 10px;
+    transition: background-color 0.3s ease;
+}
+
+#Help_Button {
+    background-color: purple;
+    
+    text-align: center;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 10px auto;
+    font-size: 30px;
+    cursor: pointer;
+    border-radius: 10px;
+    padding: 7.5px 15px;
+    margin-bottom: 10px;
+    transition: background-color 0.3s ease;
+}
+    
+#video-background {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    object-fit: cover; 
+    z-index: -2; 
+    opacity: 0.2;
+}
+    
+#content {
+    position: relative;
+    z-index: 1; /* Place the content above the video */
+    color: #fff; /* Set text color for better visibility */
+    padding: 20px;
+
```

Comment:
I dont think we have anything with an id of content in the html file. So we can get rid of this block!